### ISSUE TITLE
installation fail

### ISSUE THREAD

**Issue by SadatRafi11** [ashwoolford/bnltk#3]
![image](https://user-images.githubusercontent.com/62064596/236949101-520c2e11-795f-4e6b-8be5-e2a445fb380e.png)

**Comment by EDkinGG** [comment]
Did you find any solution of this error?

**Comment by parthokr** [comment]
@EDkinGG 
@SadatRafi11 
It seems like the issue is originating from scikit-learn. As there is no setup.py file, I am assuming this wheel file uses sklearn as package name while installing via pip.
Now this should be scikit-learn.

I have forked this repo and updated the dependencies here [bnltk-revamped](https://github.com/parthokr/bnltk-revamped).
Now you should be able to install this package by running
`pip3 install bnltk-revamped==0.1.0` this command


TLDR: run `pip3 install bnltk-revamped==0.1.0`

**Comment by asraf-patoary** [comment]
@SadatRafi11 @EDkinGG Thanks for reaching out. The package has been unmaintained for a long time. The error you were getting is about the old version of sklearn. They changed the package name from sklearn to scikit-learn. 

Now we have released bnltk v0.7.7 which should resolve your issue.
